content/en/docs/concepts/sampling/index.md

```diff
@@ -66,11 +117,9 @@ The upsides to head sampling are:
 - Can be done at any point in the trace collection pipeline
 
 The primary downside to head sampling is that it is not possible to make a
-sampling decision based on data in the entire trace. This means that head
-sampling is effective as a blunt instrument, but is wholly insufficient for
-sampling strategies that must take whole-system information into account. For
-example, it is not possible to use head sampling to ensure that all traces with
-an error within them are sampled. For this, you need Tail Sampling.
+sampling decision based on data in the entire trace. For example, you cannot
+ensure that all traces with an error within them are sampled with head sampling
+alone. For this situation and many others, you need Tail Sampling.
```

Comment:
```suggestion
alone. For this situation and many others, you need tail sampling.
```

Unless there's a powerful reason for capitalizing `Tail Sampling`, lower case all instances. Same for other sampling types.